packages/site/src/components/GuideContent.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
```

Comment:
You need to run `npm run format` as none of these new files have the license header.